joomla/joomla-cms
# [4.0] Notices in articles layouts

### Summary of Changes
Preventing Notices for articles `image_intro_caption` and `image_fulltext_caption` when these fields are empty

### Testing Instructions
Display a blog of articles for which you have not set any image captions whether for full or intro text.
Display a full single article.

In both cases you will get a Notice
`Trying to get property of non-object in ROOT//layouts/joomla/content/intro_image.php on line 16`
and
`Trying to get property of non-object in ROOT//layouts/joomla/content/full_image.php on line 14`

Patch and test again.

NOTE: When we have captions, they display on top of the image and not below. This PR does not correct this.
